scipy/spatial/transform/_rotation.pyx

```diff
@@ -1883,7 +1907,26 @@ cdef class Rotation:
             return np.asarray(rotvec)
 
     @cython.embedsignature(True)
-    def _compute_euler(self, seq, degrees, algorithm):
+    def _compute_euler(self, seq, degrees, algorithm, *, suppress_warnings):
+        """
```

Comment:
Should docstrings be written for private internal functions in Scipy? (the user should not call `_compute_euler` directly).

Reply:
I don't see any reason why not, it can be useful for any dev modifying the function (me) instead of tracking what it does or where this is called from.

It won't render in the html docs if its your concern, or I can use `#` instead of `"""`, one advantage of `"""` is that some tools can still check this docstring for correctness even if not rendered by sphinx.